dataactvalidator/scripts/initialize.py

```diff
@@ -62,7 +61,6 @@ def loadValidator():
 
 def setupDB():
     setupValidationDB()
-    setupStagingDB()
```

Comment:
Where will the staging db be created now? I've been setting up the app by [running the broker and validator initialize.py scripts](https://github.com/fedspendingtransparency/data-act-broker-backend/blob/db-547-staging-field-names/doc/CONTRIBUTING.md#initialize-broker-backend-applications), and when I do that in this branch, no staging db is created.